src/simulators/matrix_product_state/matrix_product_state_internal.cpp

```diff
@@ -1708,7 +1709,6 @@ void MPS::initialize_component_internal(const reg_t &qubits,
 }
 
 void MPS::reset(const reg_t &qubits, RngEngine &rng) {
-  move_all_qubits_to_sorted_ordering();
```

Comment:
Just verifying that you're removing this line because you've reached the conclusion that `reset` will work properly without it, and not by mistake

Reply:
I thought it could be removed, but now I am having second thoughts. I will revisit this point.